Product/Production/Common/CONNECTCoreLib/src/main/java/gov/hhs/fha/nhinc/properties/HibernateUtilFactory.java

```diff
@@ -0,0 +1,145 @@
+/**
+ *
+ */
+package gov.hhs.fha.nhinc.properties;
```

Comment:
just a weird package to put HibernatUtilFactory class .. could create one as gov.hhs.fha.nhinc.persistence or use your best judgement and resuse any other package